test/lib/BedrockTester.cpp

```diff
@@ -520,80 +445,34 @@ vector<SData> BedrockTester::executeWaitMultipleData(vector<SData> requests, int
 SQLite& BedrockTester::getSQLiteDB()
 {
     if (!_db) {
-        _db = new SQLite(_dbName, 1000000, 3000000, -1);
-    }
-    if (autoRollbackEveryDBCall) {
-        _db->rollback();
+        _db = new SQLite(_args["-db"], 1000000, 3000000, -1);
     }
     return *_db;
 }
 
 string BedrockTester::readDB(const string& query)
 {
-    return getSQLiteDB().read(query);
+    SQLite& db = getSQLiteDB();
+    db.beginTransaction();
+    string result = db.read(query);
+    db.rollback();
+    return result;
 }
 
 bool BedrockTester::readDB(const string& query, SQResult& result)
 {
-    return getSQLiteDB().read(query, result);
-}
-
-int BedrockTester::getServerPort() {
-    return _serverPort;
-}
-
-int BedrockTester::getNodePort() {
-    return _nodePort;
-}
-
-int BedrockTester::getControlPort() {
-    return _controlPort;
-}
-
-bool BedrockTester::waitForState(string state, uint64_t timeoutUS, bool control) {
-    return waitForStates({state}, timeoutUS, control);
-}
-
-bool BedrockTester::waitForStates(set<string> states, uint64_t timeoutUS, bool control)
-{
-    STable json;
-    uint64_t start = STimeNow();
-    while (STimeNow() < start + timeoutUS) {
-        try {
-            json = SParseJSONObject(executeWaitVerifyContent(SData("Status"), "200", control));
-            auto it = states.find(json["state"]);
-            if (it != states.end()) {
-                return true;
-            }
-            // It's still not there, let it try again.
-        } catch (...) {
-            // Doesn't do anything, we'll fall through to the sleep and try again.
-        }
-        usleep(100'000);
-    }
-
-    // we timed out, let's get some debugging info
-    cout << "waitForStates() timed out at " << STimeNow() << " waiting for states " << SComposeList(states) << ". Started waiting at " << start << " Most recent status: " << SComposeJSONObject(json) << endl;
-
-    return false;
-}
-
-STable BedrockTester::getStatus(bool control) {
-    // FYI: sometimes the status command doesn't return with every key/value expected
-    return SParseJSONObject(executeWaitVerifyContent(SData("Status"), "200", control));
+    SQLite& db = getSQLiteDB();
+    db.beginTransaction();
```

Comment:
Because `getSQLiteDB` used to have a *really* unintuitive behavior that you can see if you look up `autoRollbackEveryDBCall` in the deleted code. It used to start and end transactions in places you would never expect to try and maintain compatibility with old auth test code. That's all been removed.